test/cri-containerd/sandbox.go

```diff
@@ -9,6 +9,33 @@ import (
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
+type SandboxConfigOpt func(*runtime.PodSandboxConfig) error
+
+func WithSandboxAnnotations(annotations map[string]string) SandboxConfigOpt {
+	return func(config *runtime.PodSandboxConfig) error {
+		if config.Annotations == nil {
+			config.Annotations = make(map[string]string)
+		}
+		for k, v := range annotations {
+			config.Annotations[k] = v
+		}
+		return nil
```

Comment:
do we need to copy them? Can we not just set config.Annotations to the annotations map passed in?

Reply:
yeah, we need to copy, since we could potentially chain multiple `WithSandboxAnnotations` opts.